assets/styles/ot-scss/_lists.scss

```diff
@@ -156,6 +156,7 @@ aside ul {
 .overview-list {
 
   li {
+     padding: $small-spacing*0.75 $small-spacing*0.60;
```

Comment:
This overwrites the right and left padding for overview-list so that all "boxes" (registration, gender, status, number of participants) can fit on one line.